setup/bundles.rst

```diff
@@ -173,7 +173,7 @@ Before diving into the specifics of the most common edge cases, the general
 recommendation is to **not rely on the Symfony Kernel version** to decide which
 code to use::
 
-    if (Kernel::VERSION_ID <= 20800) {
+    if (Kernel::VERSION_ID < 20800) {
```

Comment:
Of course there are situations when you can need both switches, like `choices_as_values`, but IMO the most frequent and encourages should be 2.7 to 2.8